standalone/standalone.py

```diff
@@ -2260,33 +2316,33 @@ def find_node_dataset_directories(base_dir: str):
 run_final_eval_op(mmlu_branch_output="{MMLU_BRANCH_SCORES_PATH}", mt_bench_branch_output="{MT_BENCH_BRANCH_SCORES_PATH}", candidate_model="{CANDIDATE_MODEL_PATH}", taxonomy="{TAXONOMY_PATH}", tasks="{DATA_PVC_SDG_PATH}", base_branch="", candidate_branch="", device=None, base_model_dir="{DATA_PVC_MODEL_PATH}", max_workers="{MAX_WORKERS}", merge_system_user_message={MERGE_SYSTEM_USER_MESSAGE}, model_dtype="{MODEL_DTYPE}", few_shots={FEW_SHOTS}, batch_size={BATCH_SIZE})
 """
 
-    if eval_type == "mt-bench":
-        init_containers = [
-            kubernetes.client.V1Container(
-                name=f"run-eval-{eval_type}",
-                image=RHELAI_IMAGE,
-                command=["/bin/sh", "-ce"],
-                args=[
-                    PYTHON_EXECUTOR.format(
-                        python_code=exec_run_mt_bench_op_command,
-                        python_main=exec_run_mt_bench_op_args.strip(),
-                    ),
-                ],
-                volume_mounts=get_vol_mount(),
-                security_context=get_security_context(),
-                env_from=[
-                    kubernetes.client.V1EnvFromSource(
-                        secret_ref=kubernetes.client.V1SecretEnvSource(
-                            name=judge_serving_model_secret
-                        )
-                    ),
-                ],
-                resources=kubernetes.client.V1ResourceRequirements(
-                    requests={"cpu": "1", "nvidia.com/gpu": nproc_per_node},
-                    limits={"cpu": "1", "nvidia.com/gpu": nproc_per_node},
-                ),
-            )
-        ]
+    eval_container = kubernetes.client.V1Container(
+        name=f"run-eval-{eval_type}",
+        image=RHELAI_IMAGE,
+        command=["/bin/sh", "-ce"],
+        args=[
+            PYTHON_EXECUTOR.format(
+                python_code=exec_run_mt_bench_op_command,
+                python_main=exec_run_mt_bench_op_args.strip(),
+            ),
+        ],
+        volume_mounts=get_vol_mount(),
+        security_context=get_security_context(),
+        env_from=[
+            kubernetes.client.V1EnvFromSource(
+                secret_ref=kubernetes.client.V1SecretEnvSource(
+                    name=judge_serving_model_secret
+                )
+            ),
+        ],
+        resources=kubernetes.client.V1ResourceRequirements(
+            requests={"cpu": "1", "nvidia.com/gpu": nproc_per_node},
+            limits={"cpu": "1", "nvidia.com/gpu": nproc_per_node},
+        ),
+    )
+
+    if eval_type == EVAL_TYPE_MT_BENCH:
```

Comment:
small code cleanliness nitpick: since we're centralizing the init container, perhaps we should also simply the `container` in the same way, since the only diff between `EVAL_TYPE_MT_BENCH` and `EVAL_TYPE_FINAL` appears to be the `arg` parameter passed in